gamestonk_terminal/portfolio_optimization/port_optimization_controller.py

```diff
@@ -0,0 +1,160 @@
+"""Portfolio Optimization Controller Module"""
+__docformat__ = "numpy"
+
+
+import argparse
+from typing import List, Set
+from datetime import datetime
+
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair, parse_known_args_and_warn
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.portfolio_optimization import port_opt_api as po_api
+from prompt_toolkit.completion import NestedCompleter
+
+class PortfolioOptimization:
+
+    CHOICES  = ["help",
+                "q",
+                "quit",
+                "clear",
```

Comment:
As you mentioned before, I wouldn't bother with the `clear` instead I would add `select` command.
So that user could do:
* select - To select list of tickers
* add - To add to the existing list of tickers

Reply:
so select would overwrite any added so far?

Reply:
Exactly :)